rules/workload-with-cluster-takeover-roles/filter.rego

```diff
@@ -0,0 +1,32 @@
+package armo_builtins
+
+deny[msga] {
+    wl := input[_]
+    start_of_path := get_beginning_of_path(wl)
+
+    msga := {
+        "alertMessage": sprintf("%v: %v in the following namespace: %v mounts service account tokens by default", [wl.kind, wl.metadata.name, wl.metadata.namespace]),
+        "packagename": "armo_builtins",
+        "alertScore": 9,
+        "alertObject": {
+            "k8sApiObjects": [wl]
+        },
+    }
+}
+
+
+get_beginning_of_path(workload) = start_of_path {
+    spec_template_spec_patterns := {"Deployment","ReplicaSet","DaemonSet","StatefulSet","Job"}
```

Comment:
CronJob, Pod?

Reply:
they have their own `get_beginning_of_path` function because they have a different path to the spec section. Doesn't matter so much in the filter.rego but this is copy paste...